CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ option(MATLAB_BINDINGS "Compile MATLAB bindings if MATLAB is found." OFF)
 option(TEST_VERBOSE "Run test cases with verbose output." OFF)
 option(BUILD_TESTS "Build tests." ON)
 option(BUILD_CLI_EXECUTABLES "Build command-line executables." ON)
+option(DISABLE_DOWNLOADS "Disable all downloading of dependencies." OFF)
```

Comment:
The wording sounds strange, couldn't we use the same you used below: "Disable downloads of dependencies during build"?

Reply:
Good point, fixed in 9b5c3f7. :+1: